dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/SearchJob.java

```diff
@@ -70,7 +76,7 @@ protected String call() throws Exception {
                                                 final List<BulkObject> objects = f.getBlobs().getObjects();
                                                 if (objects != null) {
                                                     if (objects.stream().findFirst().isPresent()) {
-                                                        treeItems.add(new Ds3TreeTableValue(bucket.getName(), f.getName(), Ds3TreeTableValue.Type.File, f.getSize(), DateFormat.formatDate(f.getCreationDate()), f.getOwner(), true, GetStorageLocations.addPlacementIconsandTooltip(objects.stream().findFirst().orElse(null))));
+                                                        treeItems.add(new Ds3TreeTableValue(bucket.getName(), f.getName(), Ds3TreeTableValue.Type.File, f.getSize(), DateFormat.formatDate(f.getCreationDate()), f.getOwner(), true, GetStorageLocations.addPlacementIconsandTooltip(objects.stream().findFirst().orElse(null).getPhysicalPlacement(),objects.stream().findFirst().orElse(null).getInCache())));
```

Comment:
In the event that there is no physical placement, this stream call will throw a null pointer exception.  Re-implement to eliminate the possibility of a NPE